<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>986</width>
    <height>508</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>40</y>
      <width>157</width>
      <height>192</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="title">
     <string>Serial</string>
    </property>
    <layout class="QGridLayout" name="gridLayout_3">
     <item row="0" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <widget class="QLabel" name="label">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Name</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboBox_name">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="1" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout_7">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Baudrate</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboBox_baudrate">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="2" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout_8">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>DataSize</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboBox_datasize">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="3" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout_9">
       <item>
        <widget class="QLabel" name="label_4">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Parity</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboBox_parity">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="4" column="0">
      <widget class="QPushButton" name="pushButton_connect">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Connect</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>11</y>
      <width>280</width>
      <height>232</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="title">
     <string>Received Data</string>
    </property>
    <widget class="QTextEdit" name="textEdit_rdata">
     <property name="geometry">
      <rect>
       <x>12</x>
       <y>28</y>
       <width>141</width>
       <height>61</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox_4">
     <property name="geometry">
      <rect>
       <x>169</x>
       <y>28</y>
       <width>99</width>
       <height>87</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Mode</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QRadioButton" name="radioButton">
        <property name="text">
         <string>1 BYTES</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioButton_2">
        <property name="text">
         <string>5 BYTES</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QLineEdit" name="lineEdit_rdata">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>120</y>
       <width>261</width>
       <height>101</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>250</y>
      <width>456</width>
      <height>196</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="title">
     <string/>
    </property>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Address</string>
       </property>
      </widget>
     </item>
     <item row="4" column="2">
      <widget class="QLineEdit" name="lineEdit_data_2">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="1" column="0" colspan="5">
      <widget class="QLineEdit" name="lineEdit_addr">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item row="4" column="3">
      <widget class="QLineEdit" name="lineEdit_data_3">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="lineEdit_data_1">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLineEdit" name="lineEdit_data_0">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Data</string>
       </property>
      </widget>
     </item>
     <item row="4" column="4">
      <widget class="QLineEdit" name="lineEdit_data_4">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="8" column="0" colspan="5">
      <widget class="QPushButton" name="pushButton_read">
       <property name="text">
        <string>Read</string>
       </property>
      </widget>
     </item>
     <item row="6" column="0" rowspan="2" colspan="5">
      <widget class="QPushButton" name="pushButton_pos">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Write</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="groupBox_5">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>0</y>
      <width>461</width>
      <height>441</height>
     </rect>
    </property>
    <property name="title">
     <string>EEPROM DATAS</string>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QTableWidget" name="tableWidget">
       <row>
        <property name="text">
         <string>0x00</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>0x01</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>0x02</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>0x03</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>0x04</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>0x05</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>0x06</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>0x07</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>0x08</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>0x09</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>0x0A</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>0x0B</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>0x0C</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>0x0D</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>0x0E</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>0x0F</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>0x10</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>0x11</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>0x12</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>0x13</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>0x14</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>0x15</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>0x16</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>0x17</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>0x18</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>0x19</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>0x1A</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>0x1B</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>0x1C</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>0x1D</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>0x1E</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>0x1F</string>
        </property>
       </row>
       <column>
        <property name="text">
         <string>0</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>1</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>2</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>3</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>4</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>5</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>6</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>7</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>8</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>9</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>A</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>B</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>C</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>D</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>E</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>F</string>
        </property>
       </column>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>986</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
